pipelines/matrix/src/matrix/pipelines/data_publication/nodes.py

```diff
@@ -0,0 +1,51 @@
+"""Nodes for the data publication pipeline.
+
+This pipeline publishes datasets to HuggingFace Hub for public access.
+"""
+
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+from pyspark.sql import DataFrame
+
+
+def publish_dataset_to_hf(data: DataFrame) -> DataFrame:
```

Comment:
can be replaced fully with `lambda x: x` in the pipeline file directly. whole file actually.

Reply:
https://github.com/everycure-org/matrix/commit/116e68ed6c8f6e1e599658ed7fe2f43f0438dec2